src/Components/Nav/index.js

```diff
@@ -8,32 +9,66 @@ export default class Nav extends Component {
     super(props);
     this.state = {
       isSearchbarOn: false,
+      isOpenAside: false,
     };
   }
 
-  searchbarOn = () => {
+  onToggleSearch = () => {
```

Comment:
```suggestion
  toggleSearchOpen = () => {
```
토글한다 + 검색 기능의 + 열림 상태를 => 이와 같이 기능 + 대상 순으로 네이밍 하는게 좋다고 생각하는데 어떠세요?